src/test/java/org/generationcp/middleware/dao/GermplasmListDAOTest.java

```diff
@@ -130,6 +131,46 @@ public void testGetGermplasmListTypes() {
 		}
 	}
 
+	@Test
+	public void testGetGermplasmPresentInOtherListsWhereListIdIsNull() {
```

Comment:
Can you put a "_" on the method name separating the test case? eg.testGetGermplasmPresentInOtherLists_ListIdIsNull
testGetGermplasmPresentInOtherLists_ListIdIsNotNull
testGetGermplasmUsedInLockedList_GermplasmIsNotInLockedList
testGetGermplasmUsedInLockedList_GermplasmIsInLockedList

I know we don't have a current standard in writing tests but we are working on coming up with unified standards for tests(to be agreed upon).